components/home/homeSection1.tsx

```diff
@@ -12,7 +12,11 @@ const HomeSection1: FunctionComponent = () => {
         subtitle="Create your Starknet Profile by claiming your stark domain and start using it everywhere in the ecosystem."
         mobileSubtitle="Step into the future with StarkNet ID. Begin by claiming your free StarkNet identity, your passport to the blockchain world. Register your unique .stark domain to transform your presence—say goodbye to the complex addresses. Elevate your StarkNet experience today."
         buttonName="Choose your domain"
-        buttonLink="https://app.starknet.id/"
+        buttonLink={
+          window.location.origin.includes("sepolia")
+            ? "https://sepolia.app.starknet.id/"
+            : "https://app.starknet.id/"
```

Comment:
no you need to use the `env` variable we use to define if the user is on testnet or mainnet

Reply:
Done! please have a look once.

Reply:
It doesn't work because we have `"true"` not `true` :) 

So you have to do:
```ts
  const isTestnet = Boolean(process.env.NEXT_PUBLIC_IS_TESTNET === "true");
```